commands/files/serialfile.go

```diff
@@ -69,7 +69,7 @@ func (f *serialFile) NextFile() (File, error) {
 	f.files = f.files[1:]
 
 	// open the next file
-	fileName := fp.Join(f.name, stat.Name())
+	fileName := fp.ToSlash(fp.Join(f.name, stat.Name()))
```

Comment:
I see a likely cause of why forward slash is required here.
Because it is used in `addNode` in `core/commands/add.go`, which uses `file.FileName()` as an argument to `InsertNodeAtPath`.

https://github.com/ipfs/go-ipfs/blob/master/merkledag/utils/utils.go#L57 assumes a forward-slash-separated path.